src/app/containers/Article/__snapshots__/index.test.jsx.snap

```diff
@@ -6,7 +6,8 @@ exports[`ArticleContainer Component 200 status code should render correctly for
   <ServiceContextProvider
     service="news"
   >
-    <PlatformContextProvider
+    <RequestContextProvider
+      bbcOrigin={null}
```

Comment:
Do we want this to be null in the snapshots?

Reply:
Done :)